lib/seoshop-api/client/order.rb

```diff
@@ -0,0 +1,62 @@
+module Seoshop
+  class Client::Order
+    class CheckoutError
```

Comment:
this errour should be inherited from StandardError i think (not sure of the three atm)